docs/debug.md

```diff
@@ -38,3 +38,22 @@ Ocasionally when creating a new issue in GitSavvy, you will be requested to prov
       
       _Note: Take care to remove any sensitive information that may be recorded in GitSavvy._
 
+# Other tools / PATH issues / Different behavure from terminal and inside Sublime
```

Comment:
behavure  -> behaviour / behavior

Also, AFAIK, the dev version of Sublime Text has addressed this issue of environmental variables.

Reply:
Thanks.

Reply:
Perhaps Sublime Text as improved the path issue. Still, I think will always be a possible issue.